### PR TITLE
Update the typography and color variables

### PR DESCRIPTION
**Pull request by Joelhelin**
We’ve worked out a new color system for primary, typography and UI palettes. 

Andreas wanted to take a look at some Bootstrap configurations before we go any further.

### CODE REVIEW

File: src/mnd-bootstrap/_variables.scss [hunk 1]
```diff
@@ -55,11 +55,11 @@
   </div>
 </div>
  */
-$gray-darker:            lighten(#000, 13.5%); // #222
-$gray-dark:              lighten(#000, 20%);   // #333
-$gray:                   #666;
-$gray-light:             lighten(#000, 60%);   // #999
-$gray-lighter:           lighten(#000, 93.5%); // #eee
+$gray-darker:           lighten(#000, 13.5%); // #222
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: src/mnd-bootstrap/_variables.scss [hunk 2]
```diff
@@ -55,11 +55,11 @@
...
+$gray-dark:             lighten(#000, 20%);   // #333
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: src/mnd-bootstrap/_variables.scss [hunk 3]
```diff
@@ -55,11 +55,11 @@
...
+$gray:                  #666;
+$gray-light:            lighten(#000, 60%);   // #999
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: src/mnd-bootstrap/_variables.scss [hunk 4]
```diff
@@ -55,11 +55,11 @@
...
+$gray-lighter:          lighten(#000, 93.5%); // #eee
```
Comment (mnd-hound) [inline src/mnd-bootstrap/_variables.scss]: Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.

**Comment by Joelhelin** [comment]
Set as WIP. :smile:

**Pull request closed without merging**